src/components/NavBar/index.js

```diff
@@ -32,19 +32,25 @@ class NavBar extends Component {
           <NavTitle>CodePurpose</NavTitle>
           <NavItems>
             <li>
-              <NavLinks href="/">Home</NavLinks>
+              <NavLinks smooth to="/#home">
+                Home
+              </NavLinks>
             </li>
             <li>
-              <NavLinks href="/">About</NavLinks>
+              <NavLinks smooth to="/#about">
+                About
+              </NavLinks>
             </li>
             <li>
-              <NavLinks href="/">Blog</NavLinks>
+              <NavLinks to="/">Blog</NavLinks>
```

Comment:
can we hide the links we aren't using yet?

Reply:
Okay, I removed these links for now.